scss/mixins/_buttons.scss

```diff
@@ -16,63 +20,45 @@
   $active-color: color-contrast($active-background),
   $disabled-background: $gray-500, // Boosted mod
   $disabled-border: $gray-500, // Boosted mod
-  $disabled-color: $white // Boosted mod
+  $disabled-color: $white, // Boosted mod
+  $inverted: false // Boosted mod
```

Comment:
Yes I've done that because I used `$border` and `$hover-color` params. Since it is not the case anymore, I can create a `$btn-focus-shadow-color` param.